mozilla-release/docshell/base/LoadContext.cpp

```diff
@@ -63,6 +64,33 @@ LoadContext::LoadContext(nsIPrincipal* aPrincipal,
   MOZ_ALWAYS_SUCCEEDS(aOptionalBase->GetUseRemoteTabs(&mUseRemoteTabs));
 }
 
+void LoadContext::SetPrivateness(bool enable) {
```

Comment:
This is just a notification of privacy flag listeners. In our case this calls tabbrowser.xml#6520